activestorage/test/models/attached/many_test.rb

```diff
@@ -746,6 +746,11 @@ def highlights
     end
   end
 
+  test "attaching blobs to a record returns the attachments" do
+    highlights = @user.highlights.attach create_blob(filename: "funky.jpg"), create_blob(filename: "town.jpg")
+    assert_instance_of ActiveStorage::Attached::Many, highlights
```

Comment:
Especially in this case, it seems worth expanding a little on the assertions here: in particular, it seems valuable to assert that the return value contains all attachments, and not just the newly-added ones that were passed in.

Reply:
I've expanded the assertions like suggested

Reply:
Thanks! I do think it'd be valuable to cover this bit:

> in particular, it seems valuable to assert that the return value contains all attachments, and not just the newly-added ones that were passed in

... which means I think we should start by pre-attaching another blob, so we can then assert that `highlights` contains all 3: the one that was already there, _and_ the two we've just added.

It would be a particularly easy thing to accidentally change about the implementation.

Reply:
Ah, I read it wrong, sorry about that. I've now expanded assertions for this case as well